tests/lib/external/snapd-testing-tools/spread.yaml

```diff
@@ -47,7 +44,25 @@ backends:
             - ubuntu-20.04-64:
                   image: ubuntu-2004-64-virt-enabled
                   storage: 20G
-                  workers: 1
+
+    openstack:
+        key: '$(HOST: echo "$SPREAD_OPENSTACK_ENV")'
+        plan: staging-cpu2-ram4-disk50
+        halt-timeout: 2h
+        groups: [default]
+        environment:
+            HTTP_PROXY: 'http://squid.internal:3128'
+            HTTPS_PROXY: 'http://squid.internal:3128'
+            NO_PROXY: 'canonical.com'
+        systems:
+            - ubuntu-20.04-64:
+                image: ubuntu-focal-20.04-amd64
+            - ubuntu-22.04-64:
+                image: ubuntu-jammy-22.04-amd64
+            - centos-9-64:
+            - debian-12-64:
+            - fedora-40-64:
+            - opensuse-15.5-64:
```

Comment:
These changes seem unrelated, is this intentional?

Reply:
yes, because when I imported snapd-testing-tools to get the changes related to centos-7 I got the changes related to openstack as well that are already merged in snapd-testing-tools

Reply:
Would this be fixed by a rebase? Or is there a separate repo for `snapd-testing-tools`?

Reply:
we have a separate repo for snapd-testing-tools which is imported by using git subtree